files/SZWEEXEC/ZWEGEN00

```diff
@@ -384,7 +406,9 @@ say 'The edit macro was invoked on each member.'
 ================================================================================
     Add the job card to each member if filled out.
 ================================================================================
-*/
+FIXME:
```

Comment:
For a future PR?

Reply:
Yes, sounds like a plan. And we should probably reconsider where to place the job card. This code supposes `zowe.setup.jobCard`, which is better than `zowe.environment`. It will require just small schema update.